kymatio/scattering3d/backend/torch_backend.py

```diff
@@ -59,10 +27,10 @@ def modulus_rotation(x, module=None):
             which is covariant to 3D translations and rotations.
     """
     if module is None:
-        module = torch.zeros_like(x)
+        module = torch.zeros_like(x)[..., 0]
```

Comment:
This is very inefficient. It'll allocate a larger array than necessary, then extract a non-contiguous view of that array.

Reply:
This is true.